library/item/item_content_registry/src/main/java/org/quiltmc/qsl/item/content/registry/mixin/AbstractFurnaceBlockEntityMixin.java

```diff
@@ -18,7 +18,9 @@
 
 import java.util.Map;
 
+import net.minecraft.nbt.NbtCompound;
```

Comment:
Definitely a checkstyle failure

Reply:
This is QSL, there are a heckton of checkstyle failures everywhere anyway

Reply:
I usually reformat on merge tbh